parsec/core/gui/main_window.py

```diff
@@ -553,19 +562,21 @@ def add_instance(self, start_arg: Optional[str] = None):
             except ValueError as exc:
                 show_error(self, _("TEXT_INVALID_URL"), exception=exc)
 
-        idx = self._get_login_tab_index()
-        if idx != -1:
-            self.switch_to_tab(idx)
-        else:
-            tab = self.add_new_tab()
-            tab.show_login_widget()
-            self.on_tab_state_changed(tab, "login")
-            self.switch_to_tab(self.tab_center.count() - 1)
-            idx = self.tab_center.count() - 1
+        # Do not open a new logging widget if the organisation is already opened
+        if (
+            action_addr
+            and isinstance(action_addr, BackendOrganizationFileLinkAddr)
+            and self.tab_center.count()
```

Comment:
why the check on `self.tab_center.count()` given we iterate on this count in `got_to_file_link` ?

(and is there actual case where we end up with no widget in `tab_center` ? I though we would at least always have a logging widget present there...)

Reply:
that's true, this check is pointless,  it has been refactored